src/components/common/header/NavBar.js

```diff
@@ -28,7 +28,7 @@ class NavBar extends Component {
             </a>
           </li>
         </ul>
-        <UserSettings isLoggedIn={false} />
+        <UserSettings isLoggedIn />
```

Comment:
where is this prop being supplied from? or is set to true for testing purposes?

Reply:
This question would be best answered by @mrSerious. When this was set to false, anybody accessing the homepage and other pages would be logged in. Whereas, our implementation stated otherwise. I changed the props value to true to suit the implementation of signup feature